backend/src/middlewares/admin-redirect.ts

```diff
@@ -0,0 +1,10 @@
+export default (config, { strapi })=> {
+    const redirects = ['/', '/index.html', '/admin', '/admin/'].map((path: string) => ({
+        method: 'GET',
+        path,
+        handler: (ctx: any) => ctx.redirect('/admin/content-manager'),
+        config: { auth: false },
+    }));
+
+    strapi.server.routes(redirects);
+};
```

Comment:
Non-blocking: I like to have new lines at the end of files. If you use VS Code you can use the `"files.insertFinalNewline" : true` rule to have it do it automatically for you.